apps/oversikt-frontend/README.md

```diff
@@ -0,0 +1,21 @@
+# testnorge-oversikt-frontend
+
+Appliaksjon for bruk av Team Dolly til å kunne genere personlige tokens til brukt i apper som ikke støtter Client Credential (service brukere).
```

Comment:
"Appliaksjon for bruk av Team Dolly til å kunne genere personlige tokens til bruk i apper som ikke støtter Client Credential (service brukere)."

Reply:
Takk!